daml-lf/parser/src/main/scala/com/digitalasset/daml/lf/testing/parser/AstRewriter.scala

```diff
@@ -342,15 +342,39 @@ private[daml] class AstRewriter(
         InterfaceMethod(name, apply(returnType))
     }
 
+  def apply(x: InterfaceCoImplements): InterfaceCoImplements =
+    x match {
+      case InterfaceCoImplements(
+            templateId,
+            methods,
+          ) =>
+        InterfaceCoImplements(
+          apply(templateId),
+          methods.transform((_, x) => apply(x)),
+        )
+    }
+  def apply(x: InterfaceCoImplementsMethod): InterfaceCoImplementsMethod =
+    x match {
+      case InterfaceCoImplementsMethod(
+            name,
+            value,
+          ) =>
+        InterfaceCoImplementsMethod(
+          name,
+          apply(value),
+        )
+    }
+
   def apply(x: DefInterface): DefInterface =
     x match {
-      case DefInterface(requires, param, choices, methods, precond) =>
+      case DefInterface(requires, param, choices, methods, precond, coImplements) =>
         DefInterface(
           requires.map(apply(_)),
           param,
           choices.transform((_, v) => apply(v)),
           methods.transform((_, v) => apply(v)),
           apply(precond),
+          coImplements.map({ case (t, x) => (apply(t), apply(x)) }),
```

Comment:
purely cosmetic

```suggestion
          coImplements.map{ case (t, x) => (apply(t), apply(x)) },
```

Reply:
ah nice, I'd copied this part from the version of `apply` for `Template`, I'll fix that one too.